src/modules/helpers/index.ts

```diff
@@ -216,7 +300,7 @@ export class HelpersModule {
   /**
    * Replaces the symbols and patterns in a credit card schema including Luhn checksum.
    *
-   * This method supports both range patterns `[4-9]` as well as the patterns used by `replaceSymbolWithNumber()`.
+   * This method supports both regex patterns `[4-9]` as well as the patterns used by `replaceSymbolWithNumber()`.
```

Comment:
```suggestion
   * This method supports both range patterns `[4-9]` as well as the patterns used by `replaceSymbolWithNumber()`.
```

While this looks like regex it isnt yet.